ports/nrf/mpconfigport.h

```diff
@@ -163,7 +163,7 @@
 #endif
 
 #ifndef MICROPY_PY_MACHINE_SOFT_PWM
-#define MICROPY_PY_MACHINE_SOFT_PWM (0)
+#define MICROPY_PY_MACHINE_SOFT_PWM (1)
```

Comment:
Are there any dependencies left to `SOFT_PWM`? Could this be turned off by default again?

Reply:
Sure